lib/net.js

```diff
@@ -1469,8 +1469,10 @@ Object.defineProperty(Server.prototype, 'listening', {
 Server.prototype.address = function() {
   if (this._handle && this._handle.getsockname) {
     var out = {};
-    this._handle.getsockname(out);
-    // TODO(bnoordhuis) Check err and throw?
+    var err = this._handle.getsockname(out);
+    if (err) {
+      throw errnoException(err, 'address');
```

Comment:
Yeah, that’s why I’ve labelled it as a major. ;) But emitting an error event a) would be semver-major too and b) doesn’t really make sense here, it’s a fully synchronous call.